src/CommandingOfficers/AW1/AW1Commander.java

```diff
@@ -0,0 +1,92 @@
+package CommandingOfficers.AW1;
+
+import CommandingOfficers.Commander;
+import CommandingOfficers.CommanderAbility;
+import CommandingOfficers.CommanderInfo;
+import CommandingOfficers.CommanderInfo.InfoPage;
+import java.util.ArrayList;
+import Engine.GameScenario;
+import Engine.UnitMods.DamageMultiplierOffense;
+import Engine.UnitMods.DamageMultiplierDefense;
+import Engine.UnitMods.UnitModifier;
+import Terrain.MapMaster;
+import Units.Unit;
+import Units.UnitDelta;
+
+public abstract class AW1Commander extends Commander
+{
+  private static final long serialVersionUID = 1L;
+  public static final InfoPage AW1_MECHANICS_BLURB = new InfoPage(
+      "COP charge gained from combat is funds damage taken and 1/4 of funds damage dealt.\n"
+    + "CO powers don't charge while a power is active.\n"
+    + "On each power activation, each star in your meter costs 1/5 more of the base star cost.\n"
+    + "This continues for 9 activations, after which the star cost settles at double the base star cost.\n"
+    );
+  public static final int AW1_STAR_VALUE = 5000;
+
+  public AW1Commander(CommanderInfo info, GameScenario.GameRules rules)
+  {
+    super(info, rules);
+    aw1Combat = true;
+  }
+
+  @Override
+  public int calculateCombatCharge(UnitDelta minion, UnitDelta enemy, boolean isCounter)
+  {
+    if( null != getActiveAbility() )
+      return 0;
+    if( minion == null || enemy == null )
+      return 0;
+
+    int guiHPLoss  = minion.getHealthDamage() / 10;
+    int guiHPDealt =  enemy.getHealthDamage() / 10;
+
+    int power = 0; // value in funds of the charge we're getting
+
+    // Add up the funds value of the damage done to both participants.
+    power += guiHPLoss * minion.unit.getCost() / 10;
+    power += guiHPDealt * enemy.unit.getCost() / 10 / 4;
+
+    return power;
+  }
+  @Override
+  public int calculateMassDamageCharge(Unit minion, int lostHP)
+  {
+    return 0;
```

Comment:
No charge from damage from other CO's powers? Sad day.

Reply:
_waves cane_ Back in my day, the only sources of map damage were Drake and Sturm.
Drake is pretty bad (80A air, he does 1 damage for 40k charge, and no fuel thing), and Sturm is the final boss.
In that light, it makes a lot of sense.